controllers/spec/function.go

```diff
@@ -55,10 +55,15 @@ func MakeFunctionService(function *v1alpha1.Function) *corev1.Service {
 
 func MakeFunctionStatefulSet(function *v1alpha1.Function) *appsv1.StatefulSet {
 	objectMeta := MakeFunctionObjectMeta(function)
-	return MakeStatefulSet(objectMeta, function.Spec.Replicas, function.Spec.DownloaderImage,
-		MakeFunctionContainer(function), makeFunctionVolumes(function), makeFunctionLabels(function), function.Spec.Pod,
-		*function.Spec.Pulsar, function.Spec.Java, function.Spec.Python, function.Spec.Golang,
-		function.Spec.VolumeMounts)
+	if !utils.EnableDistrolessContainer {
```

Comment:
Checks the feature flag.The modification on the existing code remains small, so it's less likely to impact the existing code execution.